scripts/benchview.py

```diff
@@ -400,7 +400,7 @@ def __run_scripts(
     #   This value should be optional, and set when applicable.
     # benchview_config['Jit'] = 'RyuJIT'
     benchview_config['Framework'] = framework
-    benchview_config['.NET Compilation Mode'] = args.dotnet_compilation_mode
+    benchview_config['.NET Compilation Mode'] = 'Tiered'
```

Comment:
Instead of hardcoding, wouldn't it make more sense to delay this particular change? OR hardcode it on the yml file?

Reply:
I think the latter makes more sense.

Reply:
I do not see a meaningful difference between the two. We already hardcode other values here. This code is going to be gone in a few weeks.